docs/3.0.0-beta.x/concepts/configurations.md

```diff
@@ -451,10 +451,12 @@ The session doesn't work with `mongo` as a client. The package that we should us
   - `enabled` (boolean): Enable or disable CRON tasks to schedule jobs at specific dates. Default value: `false`.
 - `admin`
   - `autoOpen` (boolean): Enable or disabled administration opening on start. Default value: `true`.
-  - `path` (string): Allow to change the URL to access the admin panel. Default value: `/admin`.
   - `watchIgnoreFiles` (array): Add custom files that should not be watched during development. See more [here](https://github.com/paulmillr/chokidar#path-filtering) (property `ignored`). Default value: `[]`.
+  - `server`
```

Comment:
These need to be more clear I think. 

both `server` and `publicPath` say to use `/admin` but which one does what.